configs/components/rubygem-ffi.rb

```diff
@@ -0,0 +1,35 @@
+component "rubygem-ffi" do |pkg, settings, platform|
+  pkg.version "1.9.6"
+
+  if platform.is_windows?
+    if platform.architecture == "x64"
+      pkg.md5sum "daf0d310a5c498906f94f8c10006fd39"
+      pkg.url "http://buildsources.delivery.puppetlabs.net/ffi-#{pkg.get_version}-x64-mingw32.gem"
+    else
+      pkg.md5sum "5a0b3f3602f65a2b7fe6569a3333a068"
+      pkg.url "http://buildsources.delivery.puppetlabs.net/ffi-#{pkg.get_version}-x86-mingw32.gem"
```

Comment:
I thought we were setting up an internal gems repository. Is this really where it would live?